source/core/rtc_adapter/RtcAdapter.cc

```diff
@@ -89,11 +76,33 @@ class RtcAdapterImpl : public RtcAdapter,
     }
     std::shared_ptr<rtc::TaskQueue> taskQueue() override { return g_taskQueue; }
     std::shared_ptr<webrtc::RtcEventLog> eventLog() override { return g_eventLog; }
+    webrtc::WebRtcKeyValueConfig* trial() override { return g_fieldTrial.get(); }
+    ControllerSendPtr rtpTransportController() override
+    {
+        return m_transportControllerSend;
+    }
+    uint32_t estimatedBandwidth() override
```

Comment:
name this as "estimatedAvailableVideoBandwidth"

Reply:
Also I have question about such kind of splitting. if estimation is for all video tracks, any reason splitting them evenly?